src/modules/facility-fixed-fee/dto/get-facility-fixed-fee-response.dto.ts

```diff
@@ -0,0 +1,105 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { ENUMS, EXAMPLES } from '@ukef/constants';
+import { DateOnlyString } from '@ukef/helpers';
+
+export type GetFacilityFixedFeeResponse = GetFacilityFixedFeeResponseItem[];
+export class GetFacilityFixedFeeResponseItem {
+  @ApiProperty({
+    description: 'The identifier of the facility.',
+    example: EXAMPLES.FACILITY_ID,
+    minLength: 10,
+    maxLength: 10,
+  })
+  readonly facilityIdentifier: string;
+
+  @ApiProperty({
+    description: 'The identifier of the portfolio.',
+    example: ENUMS.PORTFOLIO.E1,
+  })
+  readonly portfolioIdentifier: string;
+
+  @ApiProperty({
+    description: 'The fixed amount to be billed to the client. If a flat amount is charged, this field is required.',
+    example: EXAMPLES.DEAL_OR_FACILITY_VALUE,
+    minLength: 0,
+    maxLength: 1e17,
```

Comment:
Is this correct to use `minLength`/`maxLength`? Isn't this the min/max value of the number, not the length of the string?